src/main/webapp/common.js

```diff
@@ -95,9 +95,14 @@ function updateAddressFromGeolocation(checkboxId, addressId) {
         }).catch((error) => geolocationError(error));
     } else {
       // clear the address box if location is not supported 
-      addressBox.value = ""; 
+      document.getElementById(addressId).value = ""; 
```

Comment:
Nit: please use single quotes for string literals - https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

```suggestion
      document.getElementById(addressId).value = ''; 
```

Reply:
I changed them.